paddle/fluid/pybind/inference_api.cc

```diff
@@ -249,13 +252,37 @@ void BindAnalysisConfig(py::module *m) {
       .def("cpu_math_library_num_threads",
            &AnalysisConfig::cpu_math_library_num_threads)
       .def("to_native_config", &AnalysisConfig::ToNativeConfig)
+      .def("enable_quantizer", &AnalysisConfig::EnableMkldnnQuantizer)
+      .def("quantizer_enabled", &AnalysisConfig::mkldnn_quantizer_enabled)
+      .def("quantizer_config", &AnalysisConfig::mkldnn_quantizer_config,
+           py::return_value_policy::reference)
       .def("set_mkldnn_op", &AnalysisConfig::SetMKLDNNOp)
       .def("set_model_buffer", &AnalysisConfig::SetModelBuffer)
       .def("model_from_memory", &AnalysisConfig::model_from_memory)
       .def("pass_builder", &AnalysisConfig::pass_builder,
            py::return_value_policy::reference);
 }
 
+void BindMkldnnQuantizerConfig(py::module *m) {
```

Comment:
Please, wrap this method with `#ifdef PADDLE_WITH_MKLDNN`

Reply:
Done